addons/microsoft_calendar/static/src/js/microsoft_calendar.js

```diff
@@ -179,40 +199,37 @@ const MicrosoftCalendarRenderer = CalendarRenderer.include({
     }),
 
     events: _.extend({}, CalendarRenderer.prototype.events, {
-        'click .o_microsoft_sync_button': '_onSyncMicrosoftCalendar',
-        'click .o_stop_microsoft_sync_button': '_onStopMicrosoftSynchronization',
+        'click .o_microsoft_sync_pending': '_onSyncMicrosoftCalendar',
+        'click .o_microsoft_sync_button_configured': '_onStopMicrosoftSynchronization',
     }),
 
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
 
     _initMicrosoftPillButton: function() {
-        this.$microsoftStopButton.css({"cursor":"pointer", "font-size":"0.9em"});
-        var switchBadgeClass = (elem) => {elem.toggleClass('badge-primary'); elem.toggleClass('badge-danger');};
-        this.$('.o_stop_microsoft_sync_button').hover(() => {
-            switchBadgeClass(this.$microsoftStopButton);
-            this.$microsoftStopButton.html("<i class='fa mr-2 fa-times'/>".concat(_t("Stop the Synchronization")));
+        // hide the pending button
+        this.$calendarSyncContainer.find('#microsoft_sync_pending').hide();
+        const switchBadgeClass = (elem) => {elem.toggleClass('badge-primary'); elem.toggleClass('badge-danger');};
```

Comment:
I'm not sure but I think we could do this: 🙂 
```suggestion
        const switchBadgeClass = (elem) => { elem.toggleClass('badge-primary').toggleClass('badge-danger'); };
```

Reply:
Same comment about passing an array with the classes, therefore calling `toggleClass` only once :slightly_smiling_face: